src/components/TopBar/TopBar.tsx

```diff
@@ -37,10 +40,11 @@ export default function TopBar(props: TopBarProps): JSX.Element {
         className={isDesktop ? undefined : classes.mobile}
         disableGutters={true}
         sx={{
+          borderTop: isAcceleratorRoute ? '8px solid #EF7047' : undefined,
```

Comment:
We could ask design to add a theme color and use that. Create another ticket to follow up before merging this in?

Reply:
Sounds good @joshbuchea. I can create a ticket for design to provide a theme color.

Reply:
Ok Michael looked into this, but wasn't sure how to make the change. We'll have to take care of this when Chudo gets back next week. In the interest of moving quickly to develop the MVP, would you be ok with making this change in a future PR?

Reply:
Oops, I didn't realize you had already approved the PR. I'm going to go ahead and merge this PR and will update this color reference in a future PR once it's available in the theme.